src/server.ts

```diff
@@ -133,10 +137,10 @@ export class AceBaseServer extends SimpleEventEmitter {
         this.app = createApp({ trustProxy: true, maxPayloadSize: this.config.maxPayloadSize });
 
         // Initialize and start server
-        this.init({ authDb });
+        this.init({ authDb, server: this.config.server, route: this.config.route });
```

Comment:
no need to pass these to `init` because they are available through `this.config`